tests/utils.sh

```diff
@@ -306,8 +307,10 @@ function invoke_shunit()
 {
   command -v shunit2 > /dev/null
   if [[ "$?" -eq 0 ]]; then
+    # shellcheck source=/dev/null
```

Comment:
Were you getting errors here? I think we excluded this error in our CI.

Reply:
Oh, I'm sorry, this was pretty naïve to be honest, I just updated the docs but had the out-of-date version of that .pre-commit-config.yaml locally